### PR TITLE
programmed_id and index deviation

### PR DESCRIPTION
**Pull request by ashpatcisco**
Programmed_id and programmed_index in NHG and NH respectively are unused in gRIBI AFT variant and are applicable only in gNMI. Adding these deviations removes them from gribi_aft.proto thereby removing any confusion on their intended use either modify messages or get responses.

@xw-g @robshakir

### CODE REVIEW

File: v1/yang/gribi-aft.yang [hunk 1]
```diff
@@ -135,4 +135,16 @@ module gribi-aft {
       over gNMI for the AFT model.";
     deviate not-supported;
   }
+  deviation "/afts/next-hop-groups/next-hop-group/state/programmed-id" {
```
Comment (robshakir) [inline v1/yang/gribi-aft.yang:138]:
```suggestion
  
  deviation "/afts/next-hop-groups/next-hop-group/state/programmed-id" {
```

File: v1/yang/gribi-aft.yang [hunk 2]
```diff
@@ -135,4 +135,16 @@ module gribi-aft {
...
+    description
+      "Next Hop Group programmed IDs are not accessible over gRIBI,
+       and rather over gNMI for the AFT model.";
+    deviate not-supported;
+  }
+  deviation "/afts/next-hops/next-hop/state/programmed-index" {
```
Comment (robshakir) [inline v1/yang/gribi-aft.yang:144]:
```suggestion
  
  deviation "/afts/next-hops/next-hop/state/programmed-index" {
```

**Review by robshakir — CHANGES_REQUESTED** [review]
Thanks for this -- just a whitespace suggestion and then LGTM.

File: v1/yang/gribi-aft.yang (hunk 1 above)
Comment (ashpatcisco) [reply v1/yang/gribi-aft.yang:139]: @robshakir  fixed. please re-look.

**Review by robshakir — APPROVED** [review]
Thanks for the changes!

**Pull request merged**